images/wkdev_sdk/jhbuild/webkit-sdk-deps.modules

```diff
@@ -64,6 +69,30 @@
     </branch>
   </meson>
 
+  <meson id="gtk4">
+    <branch repo="gitlab.gnome.org"
+            checkoutdir="gtk4"
+            module="GNOME/gtk.git" tag="4.15.6"/>
+  </meson>
+
+  <meson id="libadwaita" mesonargs="-Dvapi=false -Dgtk_doc=true">
+    <branch repo="gitlab.gnome.org"
+            module="GNOME/libadwaita.git" tag="1.6.beta"/>
+    <dependencies>
+      <dep package="libyaml"/>
+      <dep package="libappstream"/>
```

Comment:
These aren't defined modules?

If our base image includes them we can just remove them from the list.

Reply:
```
  <systemmodule id="libyaml">
    <pkg-config>yaml-0.1.pc</pkg-config>
    <branch repo="system"/>
  </systemmodule>
    
  <systemmodule id="libappstream">
    <pkg-config>appstream.pc</pkg-config>
    <branch repo="system"/>
  </systemmodule>
```

?

Reply:
NOTE: The libadwaita definition was already there, I just moved it around -- these dependencies were already there. I can try removing them.

Reply:
Works fine with removing them, thanks for the suggestion.